packages/devextreme/scss/widgets/material/textArea/_index.scss

```diff
@@ -10,6 +10,10 @@
 
 // adduse
 
+.dx-textarea {
```

Comment:
This style is better place after mixin 'dx-textarea(' call. Inside mixin you can have the same selector `.dx-textarea {}` and in this case it's will more priority that yours.

Reply:
Correction: [Commit](https://github.com/DevExpress/DevExtreme/pull/26885/commits/8c3cd0d674579d15406941118b5f92d97ddecb40)